takwimu/templates/takwimu/_layouts/_base.html

```diff
@@ -77,7 +77,27 @@
   <!-- Bootstrap: Enable Tooltips, Popovers -->
   <script>
     $(function () {
-      $('[data-toggle="tooltip"]').tooltip()
+      $('[data-toggle="tooltip"]').tooltip();
+
+      $('[data-toggle="popover"]').popover({
+          container: 'body',
+          html: true
+      }).on('shown.bs.popover', function (eventShown) {
+          var $popup = $('#' + $(eventShown.target).attr('aria-describedby'));
+          $popup.find('button.close').click(function (e) {
+              $popup.popover('hide');
+          });
+          // Reset filters on popover
+          $popup.find('#reset-filter').click(function(e){
+            var _parent = $(this).attr("data-parent");
+            $("#results-table tr#result").show();
+            $("input[data-parent="+_parent+"]:checkbox:checked").prop("checked", false);
+            
+            
+            $("#"+_parent+" button .select-item").removeClass("d-none").addClass("d-flex");            
+            $("#"+_parent+" button .filter-badge").removeClass("d-flex").addClass("d-none");
+          });
+      });      
```

Comment:
Won't this trigger for all popover implementations on the site? Selector should be limited to the element being manipulated in this custom implementation.

Reply:
@DavidLemayian 
I have pushed an update where i separated the reset filter action to it's component, this way it won't be triggered in other popover implementation